TheWarWithin/HunterBeastMastery.lua

```diff
@@ -1974,6 +1991,15 @@ spec:RegisterAbilities( {
         gcd = "spell",
         school = "physical",
 
+        cycle = function()
```

Comment:
Needs to return and aura or nil.  We don't really have a way to cycle targets based on health percent right now.

`function() return if talent.a_murder_of_crows.enabled then return "a_murder_of_crows" end end,`

This format gets you the aura name when AMOC is talented and otherwise returns no value (nil).

Reply:
Okay, something to mark down for later re: execute

Reply:
Also my editor yells at me for this format `function() return if talent.a_murder_of_crows.enabled then return "a_murder_of_crows" end end,`